ykrt/src/mt.rs

```diff
@@ -573,6 +572,27 @@ impl MT {
             let is_tracing = mtt.is_tracing();
             match loc.hot_location() {
                 Some(hl) => {
+                    if is_tracing {
+                        if let MTThreadState::Tracing {
+                            frameaddr: tracing_frameaddr,
+                            hl: tracing_hl,
+                            ..
+                        } = mtt.peek_tstate()
+                        {
+                            if frameaddr != *tracing_frameaddr {
+                                // ...but no longer in the frame we started in, so we need
```

Comment:
The `...` suggests this is a continuation of a previous comment, but I can't seem to find that previous comment.

Reply:
Oops! Fixed in f9dbd81aacecdce6052d5ccb73a645cbe804da0f.